Assets/Scripts/ScriptableObjects/PlayerInformation.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using UnityEngine;
+
+namespace ScriptableObjects
+{
+    public class PlayerInformation : MonoBehaviour, Damagable
+    {
+        public PlayerInformation(double maxHealth)
+        {
+            this.maxHealth = maxHealth;
+        }
+
+        public double maxHealth { get; }
+
+        public double health { get; set; }
+
+        /** The full capacity of the shield */
+        public double shieldCapacity;
+
+        /** The remaining health of the shield */
+        public double shieldHealth;
+
+        /** Controls how much incoming damage ill be absorbed by the shield */
+        private double damageResistance;
+
+        public void removeHealth(double value)
```

Comment:
Context seems to imply that this should be an override of `Damagable.AddHealth(double)`:

`public void removeHealth(double value)` -> `public override void addHealth(double value)`

Please double check, let me know if this is intentional.